Assets/Scripts/Constants/Textures.cs

```diff
@@ -0,0 +1,56 @@
+using System.Collections;
```

Comment:
I totally didn't mean to commit this... I was messing around for a bit, but this is future work...
I could remove it, but I'd have to recommit ;-;

Reply:
All good! If it's future work, then let's keep it in